hack/verify-crds-kind.sh

```diff
@@ -67,6 +66,63 @@ for CHANNEL in experimental standard; do
   kubectl delete -f "config/crd/${CHANNEL}/gateway*.yaml"
 done
 
+# Temporary workaround for https://github.com/kubernetes/kubernetes/issues/104090
+sleep 8
+
+## Check using example YAMLs as well
+## with _only_ CEL validation
+
+
+for CHANNEL in experimental standard; do
```

Comment:
This whole block is duplicated, yes, but I didn't want to pull this into a bash function because it's more complex that way, and the webhook section has a limited lifespan.